ebpf-profiling.graphqls

```diff
@@ -30,6 +30,15 @@ input EBPFProfilingTaskFixedTimeCreationRequest {
     targetType: EBPFProfilingTargetType!
 }
 
+# The request of eBPF network topology profiling ondemand task
+input EBPFProfilingOnDemandNetworkTopologyTaskRequest {
+    # define which processes under the service instance need to be profiling
+    instanceId: String
+    # define which eBPF network topology profiling task need to be extension the duration of profiling
+    # Use the task id when the task is created through instance id
+    taskId: String
```

Comment:
Why does the request include ID?

Reply:
The keep-alive mechanism is required on the page to keep the task continuous, so we need to pass in the task ID or instance ID to extend the task duration or create it.

Reply:
Please make keepAlive mechanism separated as a new interface.

Reply:
Added.